docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Creating a Ceph NFS cluster
 
-If you use the Ceph via NFS backend with {rhos_component_storage_file_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {osp_prev_long} {osp_prev_ver}.
+If you use the Ceph via NFS backend with {rhos_component_storage_file_first_ref}, prior to adoption, you must create a new clustered NFS service on the Ceph cluster. This service will replace the standalone, pacemaker-controlled `ceph-nfs` service that was used on {rhos_prev_long} {rhos_prev_ver}.
```

Comment:
I don't see the `rhos_component_storage_file_first_ref ` attribute in the current 18.0 branch

Reply:
It's not there yet. I need to copy the attributes I used upstream to the downstream adoption-attributes.adoc.